openslides/core/views.py

```diff
@@ -35,6 +36,14 @@ class IndexView(utils_views.View):
     to the custom staticfiles directory. See STATICFILES_DIRS in settings.py.
     """
 
+    @classmethod
```

Comment:
I used this way: https://docs.djangoproject.com/en/1.7/topics/class-based-views/intro/#mixins-that-wrap-as-view